<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>331</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/newPrefix/caixa.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="finalizarButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>500</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Finalizar</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>621</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QLineEdit" name="volumeText">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="larguraText">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Altura:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>cm</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>cm</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="dataText">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>cm3</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Largura:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="comprimentoText">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Volume</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="alturaText">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Comprimento:    </string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>cm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Resultados:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
